bbc/simorgh
# Disable script-switch button in Next.JS app

Related to https://jira.dev.bbc.co.uk/browse/WSTEAM1-414 and https://jira.dev.bbc.co.uk/browse/WSTEAM1-445

Overall changes
======
- Disables the script switch button in the header if the app is Next.JS. This is because Next does not work with `react-router-dom`, which we use in the script switch button to determine the button link URL
- This is fine to disable for now, but will need to revisit in https://jira.dev.bbc.co.uk/browse/WSTEAM1-445 to have a more generic approach that works for both the Express and Next.JS apps.

Testing
======
1. Visit http://localhost:7081/serbian/new_live/c003pmmldygt/lat and ensure that the page renders correctly
2. Visit http://localhost:7080/serbian/cyr/23279016 and confirm that the script switch button in the header continues to show as normal

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
